packages/python/src/components/MemberExpression.tsx

```diff
@@ -0,0 +1,403 @@
+import {
+  Children,
+  childrenArray,
+  computed,
+  For,
+  isComponentCreator,
+  OutputSymbol,
+  reactive,
+  ref,
+  Refkey,
+  Show,
+  takeSymbols,
+  ToRefs,
+  useBinder,
+} from "@alloy-js/core";
+
+export interface MemberExpressionProps {
+  children: Children;
+}
+
+interface PartDescriptorWithId extends PartDescriptorBase {
+  id: Children;
+}
+
+interface PartDescriptorWithIndex extends PartDescriptorBase {
+  index: number;
+}
+interface PartDescriptorBase {
+  accessStyle: "dot" | "bracket";
+  hasChildren: boolean;
+  mapAccess: boolean;
+  args?: Children[];
+}
+
+type PartDescriptor = PartDescriptorWithId | PartDescriptorWithIndex;
```

Comment:
This needs to handle attribute, subscription and calls:
```
interface AttributeDescriptor {
  type: "attribute";
  name: Children;
  hasChildren: boolean;
}

interface SubscriptionDescriptor {
  type: "subscription";
  expression: Children;
  hasChildren: boolean;
}

interface CallDescriptor {
  type: "call";
  args: Children[];
  hasChildren: boolean;
}
```

Then you change the compute method to handle these correctly.

Reply:
Changed.